Util.hs

```diff
@@ -149,6 +172,10 @@ run name tool opts = do
             copyFile (name ++ "-fbuild.py") "fbuildroot.py"
             system_ $ "fbuild -j" ++ show p ++ " " ++ target ++ " > " ++ devNull
             removeFile "fbuildroot.py"
+        Gup -> do
+            createSymbolicLink (name ++ "-gup") "gup"
```

Comment:
The symbolic link command is unlikely to work on Windows. I suggest a simple `copyFile` here, which should be sufficient.